seismicpro/src/gather.py

```diff
@@ -91,19 +91,69 @@ def shape(self):
         return self.data.shape
 
     def __getitem__(self, key):
-        """Select gather headers by their names.
+        """Either select gather headers by their names or create a new `Gather` with specified traces and samples
```

Comment:
Please specify explicitly that in case headers - np.array returned, not dataframe(as expected)
```suggestion
        """Either select gather headers values by their names or create a new `Gather` with specified traces and samples
```